docs/development.md

```diff
@@ -95,7 +95,6 @@ pip3 install virtualenv
 virtualenv --python=python3 .env 
 source .env/bin/activate
 pip3 install -r requirements.txt --require-hashes
-pip3 install -e .
 python3 setup.py install # also compiles the babel translation-files
```

Comment:
I would have thought that the solution here is to remove the `python3 setup.py install` but keep `pip3 install -e .`?! After all the compilation of the babel translation-files is not necessary for a dev-env, imho?!

Reply:
Yes it turns out that keeping `pip3 install -e .`  without `python3 setup.py install` also works (I didn't realize that).   I would prefer however build instructions that include babel instructions.  What if someone tests translation issues?

Reply:
Plus one for `pip3 install -e . `- works much more reliably. @relativisticelectron if you find someone who tests translation issues I will personally guide this person through his dev setup ;-)